2024/incoming/perl-version-bumper.pod

```diff
@@ -0,0 +1,619 @@
+ancient, unmaintained, scripts and modules. Even if they load some
+feature bundle (via C<use VERSION>), any deprecated feature they might
+use will also be (retroactively) included in that bundle, preserving
+backward compatibitlity.
+
+This is how the features C<indirect>, C<multidimensional> and
+C<bareword_filehandles> came to be "removed" from later versions of Perl.
+
+This is the strategy the L<Perl Steering Council|perlgov/The Steering Council>
+has chosen to best stretch the language between the continuous
+introduction of new features and the preservation of backwards
+compatibility. For the record, many Perl 4 (which is actually Perl 3,
+from late 1989) scripts still run fine with perl 5.40.
+
+=head2 Why pick a Perl version?
+
+=head3 Line 0 semantics
+
+When C<perl> compiles Perl code, before it even reads the first byte of
+source code, it is in "Perl v5.8 compatibility mode". The C<:default>
+bundle is implicitly loaded (it only contains features that are
+backward-compatible with traditional Perl).
+
+This means that the "sensibly-written code from 20 years ago" mentioned
+above is very likely to just run unmodified, and simply work.
+
+As explained above, unguarded syntactic features are available from
+line 0.
+
+=head3 Line 1 semantics
+
+We've seen that C<use VERSION> automatically loads the corresponding
+feature bundle (and associated builtins). This single line of code enables
+all official features and disables all deprecated features for the given
+version of the Perl language.
+
+In other words, putting a C<use VERSION> line at the top (line 1) of a
+Perl script or module I<declares> which version of the I<Perl language>
+the code that follows is written under. And because the effect is lexical,
+a script can load other modules that declare they were written against
+a different version of the language, and everything works transparently.
+
+I strongly believe that C<use VERSION> should be the I<first line>
+of any Perl code.
+
+=head3 Declaring a baseline of the Perl language
+
+For decades, the first recommendation made to Perl beginners and people
+asking for help on a Perl forum was to add C<use strict; use warnings;>
+at the beginning of their code. This made Perl more helpful, as it
+would point to likely errors in programming (like undeclared variables
+or undefined values).
+
+Since Perl v5.36, these two statements are implicitly included via
+C<use VERSION>. It also enables the official features of that version
+of the language, and disables the deprecated ones.
+
+Therefore, C<use VERSION> helps define a good I<baseline> for the Perl
+language, so that the compiler can fully understand the code that follows.
+It's of course possible to include experimental features, or re-enable
+deprecated features (the latter is really not recommended), to fine-tune
+the specific dialect of Perl in which the code is written.
+
+Always use a I<stable> version number with C<use VERSION>. The point is to
```

Comment:
I think "stable" needs explaining.  That said, I don't know this paragraph is adding much.

Reply:
I dropped it.